data/mods/MindOverMatter/mutations/traits.json

```
@@ -123,7 +123,8 @@
     "points": 2,
     "description": "Who needs weapons?  Thanks to your biokinesis, you deal more melee damage while fighting with your bare hands.  This damage improves as your unarmed skill increases.",
     "starting_trait": false,
-    "valid": false
+    "valid": false,
+    "enchantments": [ "KI_STRIKE" ]
```

#73412 exist, no reason to update ki strike here
```suggestion
    "valid": false
```

---

Thanks for the heads-up!